config/routes.rb

```diff
@@ -21,4 +21,6 @@
   delete '/pets/:id', to: 'pets#destroy'
 
   delete '/shelters', to: 'shelters#destroy'
+
+  get '/users/:id', to: 'users#show'
```

Comment:
Maybe something to norm on moving forward: should we be more specific with our :id tags? So for this, change to :user_id? Not a big deal either way.